docs/themes/oso-tailwind/layouts/partials/header.html

```diff
@@ -2,12 +2,16 @@
     class="static pin-t lg:sticky lg:z-50  top-0 pin-t h-8 w-full max-w-8xl mx-auto bg-primary flex flex-none items-center">
     <div id="header-page-title" class="flex-grow">
         <p class="text-white font-base text-sm  text-center truncate">
-            Want to get started? Jump into <a class="underline" href="https://join-slack.osohq.com">our Slack</a> and an engineer from the core team can help you out.
+            {{ if eq $.Page.Language.Lang "python" }}
+                Using SQLAlchemy? Check out <a class="underline" href="{{ ref . "/new-roles" }}">early access</a> to the next version of our SQLAlchemy integration.
```

Comment:
FYI @samscott89 changed the top header text if we're in Python to point to early access.